packages/server/lib/util/ci_provider.js

```diff
@@ -132,6 +132,15 @@ const _detectProviderName = () => {
   })
 }
 
+// useProvided environment variables are used to allow users to define their own
+// values should the CI provider not have an existing or correct mapping from the list below.
+const _userProvidedProviderCiParams = () => {
```

Comment:
Do docs exist for these Cypress-specific envs?

Reply:
Not yet, not really sure on where this would end up going. Any clues?

Reply:
I would assume somewhere under the cloud docs: https://docs.cypress.io/guides/cloud/introduction#Integrate-with-source-control-providers

Unless there are separate docs we share when users use the cloud. It might also be worth adding in the cloud FAQ: https://docs.cypress.io/faq/questions/cloud-faq on when and how these could be used if data is missing in the cloud for their CI provider. As of now I'm not sure how these would map out but docs should explain that.

Reply:
Added @ https://github.com/cypress-io/cypress-documentation/pull/4949